Source/Widgets/CabbageWidgetData.cpp

```diff
@@ -310,6 +310,7 @@ void CabbageWidgetData::setCustomWidgetState (ValueTree widgetData, String lineO
         switch (HashStringToInt (identifier.toStdString().c_str()))
         {
             //======== strings ===============================
+            case HashStringToInt ("base64"):
```

Comment:
I'm not sure we need a new identifier here. I think it is better to add base64 support to the file identifier as we discussed on the forum?

Reply:
You're right. Your approach is better than mine as we discussed on the forum